daemons/attrd/attrd_cib.c

```diff
@@ -477,17 +477,12 @@ attrd_write_attribute(attribute_t *a, bool ignore_delay)
     GHashTable *alert_attribute_value = NULL;
     int rc = pcmk_ok;
 
-    CRM_CHECK(the_cib != NULL, return);
```

Comment:
Whoops, I see that I broke this in 8d3fd2b. At a low priority, this function could use some refactoring. The logic that depends (directly or indirectly) on whether we're in stand-alone mode is too spread-out for my liking.

I believe my thought at the time was "well we use `the_cib` later outside of this `!stand_alone && !a->is_private` block, so we should check it at the very top." But we don't actually get to a place where we use `the_cib` unless we already entered the `!stand_alone && !a->is_private` block.